dashboard/pom.xml

```diff
@@ -95,6 +95,7 @@
                 </executions>
                 <configuration>
                     <workingDirectory>src/main/webapp</workingDirectory>
+                    <npmRegistryURL>http://jfrog.cloud.qiyi.domain:80/api/npm/npm/</npmRegistryURL>
```

Comment:
Is this for debugging purposes? You forgot to delete it, right?

Reply:
Oh yes